locations/spiders/thrifty_foods_ca.py

```
@@ -1,68 +1,33 @@
     name = "thrifty_foods_ca"
-    item_attributes = {"brand": "Thirty Foods", "brand_wikidata": "Q7798140"}
+    item_attributes = {"brand": "Thrifty Foods", "brand_wikidata": "Q7798140"}
     allowed_domains = ["www.thriftyfoods.com"]
-    start_urls = (
-        "https://www.thriftyfoods.com/api/en/Store/get?Latitude=48.45423&Longitude=-123.359205&Skip=0&Max=60000",
-    )
+    sitemap_urls = ["https://www.thriftyfoods.com/sitemap/stores/sitemap.xml"]
+    sitemap_rules = [(r"/stores/\d+-", "parse_sd")]
+    wanted_types = ["GroceryStore"]
+    requires_proxy = True
+    custom_settings = {"USER_AGENT": BROWSER_DEFAULT}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)